### DISCUSSION TITLE
WebhookRequestHandler

### DISCUSSION THREAD

**Discussion by AlexSerb-RU** [aiogram/aiogram#634] [Q&A]
Hi, I'm writing a bot using your library and using built-in webhooks. I have a need to process POST and GET requests not only from telegram, but also from other sites, in search of an answer to the question "how to do this?" I came across a vague mention of "WebhookRequetsHandler", it seemed to me that this was exactly what I needed, I tried to find more information about its application, but my attempts were in vain. I ask you to give more information or a link to the information, and I will be eternally grateful for an example of use.

**Comment by Olegt0rr** [comment]
@AlexSerb-RU,
https://github.com/aiogram/aiogram/blob/dev-2.x/examples/separate_api_route_example.py

**Reply by NemchenkoK** [reply]
Hello. I ran into the same problem. I adapted the code from the example you link to. As a result, nothing happened. The bot didn't start. And a request to /bot causes an error in aiohttp itself (maybe because bot didn't start). I ask you to give more information or a link to the information, and I will be eternally grateful for a help.
